components/header.tsx

```diff
@@ -27,13 +28,13 @@ export function Header() {
     }
 
     return (
-        <header className="isolate sticky top-0 bg-vscode-background-light dark:bg-vscode-background-dark  z-50">
-            <div className="border-b-2 border-orange-500">
+        <header className="isolate sticky top-0 bg-vscode-background-light dark:bg-vscode-background-dark  z-20">
```

Comment:
why did we change z-index here?
changing these values may have unintended consequences, this should be the last resort

Reply:
It is the last resort, the fixed value from decoding kept coming above the banner

Reply:
mm, `z-20` is lower than `z-50`, therefore `z-50` would still work as intended